hack/README.md

```diff
@@ -40,8 +40,10 @@ The standard workflow is performed inside a helper container to normalize the bu
     - `build-importer`: compile cdi-importer binary
     - No `build-cloner` target exists as the code is written in bash
 - `test`: execute all tests
-    - `test-unit`: execute all tests under `./pkg`
-    - `test-functional`: execute all tests under `./tests`
+    - `test-unit`: execute all tests under `./pkg/`
+    - `test-functional`: execute functional tests under `./tests/`. Reads from `TEST_ARGS` to pass through flags to tested code
+- `build-functest-image-init`: build the init container for the testing file server
+- `build-functest-image-http` build the http container for the testing file server
```

Comment:
ditto

Reply:
Sorry, not sure what your referring to here.